src/renate/updaters/experimental/gdumb.py

```diff
@@ -44,6 +44,7 @@ def __init__(
     ) -> None:
         super().__init__(
             memory_size=memory_size,
+            batch_memory_frac=-1.0,  # Dummy value
```

Comment:
Should this raise an error?

Reply:
Will add a check for to be in [0, 1] and then use 0.0 as a dummy value.